### ISSUE TITLE
Error en External API al borrar regla

### ISSUE THREAD

**Issue by juacas** [UNIMOODLE/moodle-local_notificationsagent#132]
Al intentar borrar una regla se llama mediante AJAX a https://moodle-pruebas.uva.es/moodle43/lib/ajax/service.php?sesskey=qD2rEg7Eyk&info=local_notificationsagent_check_rule_context

Y se produce este error:
![image](https://github.com/user-attachments/assets/0fc68b88-894a-45ae-afe3-165a3098237d)

Que proviene del error de código:

{
    "error": "Excepci\u00f3n - Class \"external_api\" not found",
    "errorcode": "generalexceptionmessage",
    "stacktrace": null,
    "debuginfo": null,
    "reproductionlink": null
}

Se produce en Moodle 4.3 así que probablemente sea debido al cambio de APIs ocurrido en v4.2.
Esto parece estar relacionado con #125 

En el proeycto Certifygen he propuesto un PR con una solución https://github.com/UNIMOODLE/moodle-mod_certifygen/pull/18/commits/6b1c7c7425b3510ef1f70cd7f6fe2bd29e9a2a58

El cambio que agrega compatibilidad está en el fichero externalcompatibility.php que hay que incluirlo en las librerías de servicios.

**Comment by fpano** [comment]
Sí, de esto éramos conscientes y estamos ya en ello. Gracias por el PR.

**Comment by fpano** [comment]
Ya hemos implementado esta solución y ya no existe el error
